pkgs/by-name/mi/min-ed-launcher/package.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildDotnetModule,
+  fetchFromGitHub,
+  git,
+}:
+buildDotnetModule rec {
+  pname = "min-ed-launcher";
+  version = "0.11.2";
+
+  src = fetchFromGitHub {
+    owner = "rfvgyhn";
+    repo = "min-ed-launcher";
+    tag = "v${version}";
+    hash = "sha256-p970rsDyqXBbxo18q/ocNRkBKBq2CwJA3tAfDicrTOE=";
+
+    leaveDotGit = true;
```

Comment:
Does this need .git to run? Looking at the upstream repo i wouldnt know why it would.

If not, please dont make it keep the .git without reason.

Reply:
Forgot to say, if its actually needed, please add some comments next to the two git lines that explain why its needed.

Reply:
See: https://github.com/NixOS/nixpkgs/pull/386645#discussion_r2018074741